gslib/commands/rpo.py

```diff
@@ -116,6 +121,48 @@ class RpoCommand(Command):
       },
   )
 
+  gcloud_storage_map = GcloudStorageMap(
+      gcloud_command={
+          'get':
+              GcloudStorageMap(
+                  gcloud_command=[
+                      'alpha', 'storage', 'buckets', 'list', _GCLOUD_LIST_FORMAT
+                  ],
+                  flag_map={},
+                  supports_output_translation=True,
+              ),
+          'set':
+              GcloudStorageMap(
+                  gcloud_command={
+                      'enforced':
+                          GcloudStorageMap(
+                              gcloud_command=[
+                                  'alpha',
+                                  'storage',
+                                  'buckets',
+                                  'update',
+                                  '--rpo',
```

Comment:
I will have to take a look at the gcloud CL for rpo. I don't think we should have boolean flags for rpo as the set command accepts strings like ASYNC_TURBO and DEFAULT. Please ping me offline the CL for rpo.